ocs_ci/ocs/ui/page_objects/alerting.py

```diff
@@ -0,0 +1,247 @@
+import logging
+import re
+import hashlib
+import requests
+from ocs_ci.utility.retry import retry
+from requests import TooManyRedirects, Timeout, HTTPError
+
+from ocs_ci.ocs.ui.page_objects.page_navigator import PageNavigator
+from ocs_ci.ocs.ui.page_objects.searchbar import SearchBar
+
+
+logger = logging.getLogger(__name__)
+
+
+class Runbook:
+    """
+    Runbook object used for checking the runbook content.
+    May be instantiated with the runbook hash value as expected result for the test;
+    If the runbook hash value is not provided, text should be provided and the hash value will be calculated
+    """
+
+    def __init__(self, text=None, runbook_hash=None):
+
+        if not text and not runbook_hash:
+            raise ValueError("Runbook text or hash value should be provided")
+
+        self.text = text
+        if not runbook_hash:
+            self.runbook_hash = hashlib.md5(self.text.strip().encode()).hexdigest()
+        else:
+            self.runbook_hash = runbook_hash
+
+    def __repr__(self):
+        """
+        get base64 hash of the Raw runbook page content
+        """
+
+        return self.runbook_hash
+
+    def __eq__(self, other):
+        """
+        Compare two instances based on their hash values
+        """
+        if isinstance(other, Runbook):
+            return self.runbook_hash == other.runbook_hash
+        return False
+
+    def __ne__(self, other):
+        """
+        Compare two instances based on their hash values
+        """
+        return not self.__eq__(other)
+
+    def check_text_content(self, headers, *args):
+        """
+        Check if the text is valid
+
+        Returns:
+            bool: True if the text is valid, False otherwise
```

Comment:
leave empty space after this. same in all places

Reply:
done.

I wonder, if this will have effect on how the method description looks in documentation? if no tha is just adding another line without much sense. Black is not arguing too..